src/Parser/Hunter/Shared/Modules/Features/FocusUsage.js

```diff
@@ -0,0 +1,323 @@
+import React from 'react';
+import { Doughnut as DoughnutChart } from 'react-chartjs-2';
+
+import SPELLS from 'common/SPELLS';
+import SPECS from 'common/SPECS';
+
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import StatisticsListBox from 'Main/StatisticsListBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import STATISTIC_ORDER from 'Main/STATISTIC_ORDER';
+
+const CHART_SIZE = 100;
+
+const BUFFER_MS = 100;
+
+const VOLLEY_FOCUS_COST = 3;
+
+const LIST_OF_BM_FOCUS_SPENDERS = [
+  SPELLS.COBRA_SHOT.id,
+  SPELLS.MULTISHOT.id,
+  SPELLS.KILL_COMMAND.id,
+  SPELLS.REVIVE_PET_AND_MEND_PET.id,
+  SPELLS.A_MURDER_OF_CROWS_TALENT_SHARED.id,
+  SPELLS.BARRAGE_TALENT.id,
+  SPELLS.VOLLEY_ACTIVATED.id,
+];
+
+const LIST_OF_MM_FOCUS_SPENDERS = [
+  SPELLS.AIMED_SHOT.id,
+  SPELLS.MARKED_SHOT.id,
+  SPELLS.REVIVE_PET_AND_MEND_PET.id,
+  SPELLS.A_MURDER_OF_CROWS_TALENT_SHARED.id,
+  SPELLS.BARRAGE_TALENT.id,
+  SPELLS.VOLLEY_ACTIVATED.id,
+  SPELLS.WINDBURST.id,
+  SPELLS.PIERCING_SHOT_TALENT.id,
+  SPELLS.BLACK_ARROW_TALENT.id,
+  SPELLS.EXPLOSIVE_SHOT_TALENT.id,
+];
+
+//code grabbed from Parser/Paladin/Holy/Modules/PaladinCore/CastBehavior.js
+
+class FocusUsage extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  focusSpenderCastsBM = {
+    [SPELLS.COBRA_SHOT.id]: {
+      casts: 0,
+      focusUsed: 0,
+      name: "Cobra Shot",
```

Comment:
Single quotes is preferred. How about using the id and pulling the name from the spells object instead? That way you could optionally make it a SpellLink.